tools/rust_analyzer/rust_project.rs

```diff
@@ -89,11 +92,13 @@ pub struct Dependency {
 }
 
 pub fn generate_rust_project(
-    sysroot_src: &str,
+    sysroot_path: &str,
+    sysroot_src_path: &str,
     crates: &BTreeSet<CrateSpec>,
 ) -> anyhow::Result<RustProject> {
     let mut project = RustProject {
-        sysroot_src: Some(sysroot_src.into()),
+        sysroot: Some(sysroot_path.into()),
```

Comment:
Also, `sysroot` is a supported flag? I'm not seeing it in https://rust-analyzer.github.io/manual.html#non-cargo-based-projects